server/app/services/applications/ProgramAdminApplicationService.java

```diff
@@ -203,11 +203,7 @@ private void sendAdminSubmitterEmail(
     String body =
         String.format(
             "%s\n%s",
-            messages.at(
-                MessageKey.EMAIL_TI_APPLICATION_UPDATE_BODY.getKeyName(),
-                applicant.id,
-                programName,
-                newStatusText),
+            statusDef.localizedEmailBodyText().get().getOrDefault(locale),
```

Comment:
Would the StatusDef contain applicant.id, programName?

Reply:
I wonder if adding the program name would help. I would go ahead and approve but I am wondering if adding programName would be beneficial. May be a good question for the product.

Reply:
This is helpful. Thanks Nick. Yeah- a programName will be super useful.